CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 - `prometheusexporter`: Prometheus fails to generate logs when prometheus exporter produced a check exception occurs. (#8949)
 - `resourcedetectionprocessor`: Wire docker detector (#9372)
 - `kafkametricsreceiver`: The kafkametricsreceiver was changed to connect to kafka during scrape, rather than startup. If kafka is unavailable the receiver will attempt to connect during subsequent scrapes until succcessful (#8817).
+- `datadogexporter`: The command in the `exporter/datadogexporter/example/example_k8s_manifest.yaml` should be `/otelcol-contrib`. (#9425).
```

Comment:
nit: I think something like this describes better what your change is doing (rather than what was wrong).

```suggestion
- `datadogexporter`: Update Kubernetes example manifest to new executable name. (#9425).
```